src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -2,7 +2,13 @@
   "version": "1.0.0-preview3-*",
   "buildOptions": {
     "keyFile": "../../tools/Key.snk",
-    "warningsAsErrors": true
+    "warningsAsErrors": true,
+    "compile": {
+      "include": [
+        "**/*.cs",
+        "../../src/dotnet/DotnetFiles.cs"
```

Comment:
This isn't about tests - it is about other CLI Tools using this library.  When they run, `private static string SdkRootFolder => Path.Combine(typeof(DotnetFiles).GetTypeInfo().Assembly.Location, "..");` will be pointing into the NuGet cache, which won't contain the .version file.  Thus when the file is attempted to be read, it will throw an exception.